src/Uno.Toolkit.UI/Behaviors/StatusBar.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using Uno.Disposables;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+#endif
+
+using XamlColor = Windows.UI.Color;
+
+namespace Uno.Toolkit.UI
+{
+	public enum StatusBarTheme { None, Light, Dark }
```

Comment:
It would be nice to add 2 additional values:
* `Auto` to follow the current application's `RequestedTheme`.
* `AutoInversed` to follow the inverse of the current application's `RequestedTheme`